x-pack/plugins/index_management/__jest__/client_integration/home/indices_tab.test.ts

```diff
@@ -326,15 +358,31 @@ describe('<IndexManagementHome />', () => {
 
       await actions.clickModalConfirm();
 
-      const requestsCount = server.requests.length;
-      expect(server.requests[requestsCount - 2].url).toBe(`${API_BASE_PATH}/indices/forcemerge`);
-      // After the index is force merged, we immediately do a reload. So we need to expect to see
-      // a reload server call also.
-      expect(server.requests[requestsCount - 1].url).toBe(`${API_BASE_PATH}/indices/reload`);
+      expect(httpSetup.post).toHaveBeenCalledWith(
+        `${API_BASE_PATH}/indices/forcemerge`,
+        expect.anything()
+      );
+      expect(httpSetup.post).toHaveBeenCalledWith(
+        `${API_BASE_PATH}/indices/reload`,
+        expect.anything()
+      );
     });
   });
 
   describe('Edit index settings', () => {
+    const indexName = 'test';
+
+    beforeEach(async () => {
+      httpRequestsMockHelpers.setLoadIndicesResponse([createNonDataStreamIndex(indexName)]);
+
+      testBed = await setup(httpSetup);
+      const { component, find } = testBed;
+
+      component.update();
+
+      find('indexTableIndexNameLink').at(0).simulate('click');
```

Comment:
can you explain why this wasn't needed before?

Reply:
Since the testBed instance does't get re-initialized in a top level`beforeEach`, this test was wrongly depending on the testBed instance that is initialized in the previous describe. So we need to specifically declare it in its own describe scope.